pom.xml

```diff
@@ -322,7 +322,7 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.13.0</version>
                 <configuration>
-                    <release>11</release>
+                    <release>21</release>
```

Comment:
Upgrading pact maven plugin will fix issues with pact publishing job as it brings in missing library

```
 <groupId>au.com.dius.pact.provider</groupId>
                <artifactId>maven</artifactId>
                <version>4.6.10</version>
```

We can also remove the below libraries. These are brought in by `pay-java-commons`
`pact-jvm-provider-junit_2.12`
`pact-jvm-consumer-junit_2.12`

Reply:
Also need to update `.java-version` file